lua/conform/formatters/sqlfmt.lua

```diff
@@ -0,0 +1,9 @@
+---@type conform.FileFormatterConfig
+return {
+  meta = {
+    url = "https://docs.sqlfmt.com",
+    description = "sqlfmt formats your dbt SQL files so you don't have to. It is similar in nature to Black, gofmt, and rustfmt (but for SQL)",
+  },
+  command = "sqlfmt",
+  args = { "$FILENAME" },
```

Comment:
From [the docs](https://docs.sqlfmt.com/getting-started/using-sqlfmt), it looks like the correct way to format with stdin/stdout is to pass a single argument: `-`